cryodrgn/protocols/protocol_base.py

```diff
@@ -36,8 +36,10 @@
 import pyworkflow.utils as pwutils
 from pyworkflow.plugin import Domain
 from pwem.constants import ALIGN_PROJ, ALIGN_NONE
-from pwem.protocols import ProtProcessParticles
-from pwem.objects import SetOfParticles
+from pwem.protocols import ProtProcessParticles, ProtFlexBase
+from pwem.objects import SetOfParticles, ParticleFlex
+
+import cryodrgn.constants as const
```

Comment:
shall we either use this import or "from cryodrgn.constants import" below for constants?

Reply:
I prefer the way proposed by @azazellochg

Reply:
Thanks @azazellochg! I have made the changes to reuse "from cryodrgn.constants import" below